ais/local/settings.py

```diff
@@ -34,14 +34,15 @@
         'NAME': 'ais_dev',
         'USER': 'ais_dev',
         'PASSWORD': 'ais_dev',
-        'HOST': '127.0.0.1', # For Vagrant
-        # 'HOST': 'db',  # For docker
+        # 'HOST': '127.0.0.1', # For Vagrant
+        'HOST': 'db',  # For docker
```

Comment:
Revert the commit of this file as Docker is depended on PR #673, make sure to now commit this file.

Reply:
Remove it from the PR, but you can have the change locally.